host/frontend/host-orchestrator/instancemanager_test.go

```diff
@@ -55,3 +64,158 @@ func TestCreateCVDInvalidRequests(t *testing.T) {
 		}
 	}
 }
+
+type FakeFetchCVDDownloader struct {
+	t       *testing.T
+	content string
+}
+
+func (d *FakeFetchCVDDownloader) Download(dst io.Writer, buildID string, accessToken string) error {
+	r := strings.NewReader(d.content)
+	if _, err := io.Copy(dst, r); err != nil {
+		d.t.Fatal(err)
```

Comment:
this downloader is not meant for testing the fail/error route, therefore the fatal call in case Copy fails. down below we have the AlwaysFailsFetchCVDDownloader downloader which test the fail/error route.